pkg/commands/options/build.go

```diff
@@ -91,14 +91,27 @@ func (bo *BuildOptions) LoadConfig() error {
 	}
 	// If omitted, use this base image.
 	v.SetDefault("defaultBaseImage", configDefaultBaseImage)
-	v.SetConfigName(".ko") // .yaml is implicit
+	const configName = ".ko"
+
+	v.SetConfigName(configName) // .yaml is implicit
 	v.SetEnvPrefix("KO")
 	v.AutomaticEnv()
 
 	if override := os.Getenv("KO_CONFIG_PATH"); override != "" {
+		path := filepath.Join(override, configName+".yaml")
+		file, err := os.Stat(path)
+		if err != nil {
+			return fmt.Errorf("error looking for config file: %w", err)
+		}
+		if !file.Mode().IsRegular() {
+			return fmt.Errorf("config file %s is not a regular file", path)
+		}
 		v.AddConfigPath(override)
 	}
 
+	if bo.WorkingDirectory == "" {
```

Comment:
Is this change necessary?

Reply:
No, looks like we had it on line 88-90 now. Removed